src/main/java/io/xcherry/core/persistence/read_request/LocalAttributeReadRequest.java

```diff
@@ -0,0 +1,66 @@
+package io.xcherry.core.persistence.read_request;
```

Comment:
I feel like it's probably not a good practice to use underscore for package name in Java. I never see that in any popular libraries

Reply:
True, making all lower case without _ is also recommended:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html